src/engine/read_json.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+#include "json.hpp"
+using json = nlohmann::json;
+
+/* Example json format
+{
+  "0": {
+    "posy": [
+      {
+        "value": 0.0,
+        "timestamp": 0,
+        "interpolation": "LINEAR"
+      },
+      {
+        "value": 20.0,
+        "timestamp": 100,
+        "interpolation": "LINEAR"
+      },
+      {
+        "value": 0.0,
+        "timestamp": 200,
+        "interpolation": "LINEAR"
+*/        
+
+void from_json(const json &j, Animation &a) {
```

Comment:
Why is this in this file? read_json seems like a generic utils file, and it's in engine, but contains animation specific stuff... move this to animation.cpp or smth